src/validations/html/htmlInspector.js

```diff
@@ -2,14 +2,36 @@ import last from 'lodash/last';
 import isNull from 'lodash/isNull';
 import Validator from '../Validator';
 
+const specialCases = {
+  li: {
+    reason: 'invalid-tag-parent',
+    parent: '<ul>, <ol>, or <menu>',
```

Comment:
Can we get the “or” of it all into the translation? Maybe `i18next` has a clever way of allowing you to construct a natural-language list out of an array?